make-rules/shared-macros.mk

```diff
@@ -864,8 +864,8 @@ QT5_BINDIR.32 = $(QT5_BASEDIR)/bin
 QT5_LIBDIR.32 = $(QT5_BASEDIR)/lib
 QT5_BINDIR.64 = $(QT5_BASEDIR)/bin/$(MACH64)
 QT5_LIBDIR.64 = $(QT5_BASEDIR)/lib/$(MACH64)
-QT5_BINDIR = $(QT5_BINDIR.$(BITS))
-QT5_LIBDIR = $(QT5_LIBDIR.$(BITS))
+QT5_BINDIR = $(or $(QT5_BINDIR.$(BITS)),$(QT5_BINDIR.64))
+QT5_LIBDIR = $(or $(QT5_LIBDIR.$(BITS)),$(QT5_LIBDIR.64))
```

Comment:
Why do we need all of these?  I'm sure these changes are trying to solve some non-problem.  Or some problem that could and should be solved differently (and easier).